lua/orgmode/org/autocompletion/omni.lua

```diff
@@ -101,7 +106,8 @@ local function omni(findstart, base)
     return -1
   end
 
-  local fetcher_ctx = { base = base, line = line }
+  local url_str = line:match('%[%[(.-)$')
+  local url = Url.new(url_str)
```

Comment:
The `Url` is all what we need inside the `fetcher` callback.

Reply:
This change breaks the manual omni autocompletion (`<C-x><C-o>`). Doing this:
```lua
  local url = Url.new((base or '') .. url_str)
```
Solved part of the issue by properly autocompleting files, but autocompleting headlines and custom ids after `::` is not working.

I tested this by disabling the cmp in my config and just testing with `<C-x><C-o>`.